sources/packages/api/src/route-controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -174,9 +175,12 @@ export class ProgramInfoRequestController extends BaseController {
    * PIR status as Declined in the student application table.
    * @param locationId location that is completing the PIR.
    * @param applicationId application id to be updated.
-   * @param payload contains the denied reason of the
-   * student application.
+   * @param payload contains the denied reason of the student application.
    */
+  @ApiUnprocessableEntityResponse({
```

Comment:
are planning to add `ApiOkResponse` in this PR, or we have a future story for this

Reply:
This PR is a refactor on many existing endpoints and services to adapt to the new reassessment DB structure. Changing the API decorators during this task will just increase the PR and make it hard to be reviewed.